MetaMask/metamask-extension
# Improve first time flow css - select action

Improves: First time flow css - select action

Explanation:  

* Previously, cards here were using `flex` and there was a need to apply a `margin-left` to fix the centring. 
* Also text was too close to the borders.

<img width="868" alt="Screenshot 2021-11-23 at 11 45 25" src="https://user-images.githubusercontent.com/922348/143011188-59380552-ec9e-45c4-9ffd-440e04190ac2.png">

This fixes it, using `grid` to properly center the cards and adding some `padding` to them. 

<img width="856" alt="Screenshot 2021-11-23 at 11 24 24" src="https://user-images.githubusercontent.com/922348/143011216-11c3bef4-5370-49f5-97de-22fb05c654e8.png">

Manual testing steps:  
  - Load the extension for the first time.
  - Navigate to the select action view.
  - Check cards are properly aligned.
